decentraland/components/presentational/LandForSaleList.test.js

```diff
@@ -6,11 +6,13 @@ describe("LandForSaleList", () => {
   it("renders the component", async () => {
```

Comment:
It's odd the snapshot for this file didn't change in this PR

Reply:
It is the desirable behavior, in my first try (only check the array) the message was shown before the first asset load. Because of that, I've introduced the `loadingInProgress` flag and the "no assets" message only be showed when loading as completed and the list is empty.
In resume, empty list + inprogress = true has the same behavior as before (blank page).

Reply:
Ah ok - I guess what surprised me was that this test was doing an expectation on the blank page as opposed to the full page. If it was doing an expectation on a blank page before, then this diff makes sense.

Reply:
It sounds like in a future PR we'll want a third placeholder rendered view for this screen that says "Loading"

Reply:
> It sounds like in a future PR we'll want a third placeholder rendered view for this screen that says "Loading"

Yes, for sure. The UX needs to be improved with more feedback to the user on that way.